jte-kotlin/src/main/java/gg/jte/compiler/kotlin/KotlinClassCompiler.java

```diff
@@ -39,7 +39,7 @@ public void compile(String[] files, List<String> classPath, TemplateConfig confi
         K2JVMCompiler compiler = new K2JVMCompiler();
 
         SimpleKotlinCompilerMessageCollector messageCollector = new SimpleKotlinCompilerMessageCollector(templateByClassName, config.packageName);
-        ExitCode exitCode = compiler.exec(messageCollector, new Services.Builder().build(), compilerArguments);
+        ExitCode exitCode = compiler.exec(messageCollector, Services.EMPTY, compilerArguments);
```

Comment:
Hi,
Is it possible to make this correction in another PR to avoid the following error :
```
java.lang.NoSuchMethodError: 'org.jetbrains.kotlin.cli.common.ExitCode org.jetbrains.kotlin.cli.jvm.K2JVMCompiler.exec(org.jetbrains.kotlin.cli.common.messages.MessageCollector, org.jetbrains.kotlin.config.Services, org.jetbrains.kotlin.cli.common.arguments.CommonToolArguments)'
```
I got this after migrating my jte project to version 2.1.0 of Kotlin